### ISSUE TITLE
Pages not marked consistently on Android and iOS

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-shield-mobile#250]
Closing a window is not marked consistently with same value. Close, Back and Cancel are used. Cancel seems OK depending on the page. But Close and Back seem to have very similar meanings. Pages should have same value (should be consistent). 

Android has a back arrow on all screens but it reads different values that iOS has written out. It should be consistently identified.

Android 8.1.0 on Samsung
iOS 13.5.1 on iPhone 7

![3](https://user-images.githubusercontent.com/14280062/85937441-9d99cf00-b8d1-11ea-9653-b0084f73b7e6.jpg)
![4](https://user-images.githubusercontent.com/14280062/85937442-9e326580-b8d1-11ea-99a6-2e47a723f106.jpg)
![5](https://user-images.githubusercontent.com/14280062/85937443-9e326580-b8d1-11ea-9648-cfd6b510bbe8.jpg)
![6](https://user-images.githubusercontent.com/14280062/85937501-1bf67100-b8d2-11ea-9ac3-52d487c49310.jpg)

**Comment by nickmly** [comment]
@JuliannaR Should all these buttons have the same text? E.g. All of them could say 'Back

**Comment by JuliannaR** [comment]
Back should be used when navigating back to the previous page, cancel should be used when we are closing an action. This is similar to issue https://github.com/cds-snc/covid-shield-mobile/issues/272#issuecomment-653777080
